### PR TITLE
Create serverinfo.js

### PR DESCRIPTION
**Pull request by BluewyDev**

### CODE REVIEW

File: src/commands/utility/serverinfo.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        generic += `**Guild ID:** ${guild.id} \n`;
+        
+        let statistics = '';
+        statistics += `**Members:** ${guild.memberCount} \n`;
+        statistics += `**Max members:** ${guild.maximumMembers} \n`;
+        statistics += `**Verified: ${guild.verified ? 'yes' : 'no'} \n`;
+        statistics += `**Partnered:** ${guild.partnered ? 'yes' : 'no'} \n`;
+        
+        let embed = new Discord.MessageEmbed()
+            .setAuthor(`Info of ${guild.name}`, guild.iconURL());
```
Comment (JulianVennen) [inline src/commands/utility/serverinfo.js:30]:
```suggestion
            .setAuthor(`Info of ${guild.name}`, guild.iconURL())
```

File: src/commands/utility/serverinfo.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        generic += `**Owner:** ${guild.owner.user.username} \n`;
+        generic += `**Owner ID:** ${guild.ownerID} \n`;
+        generic += `**Created:** ${guild.createdAt.toDateString()} \n`;
+        generic += `**Region:** ${guild.region} \n`;
+        generic += `**Guild ID:** ${guild.id} \n`;
+        
+        let statistics = '';
+        statistics += `**Members:** ${guild.memberCount} \n`;
+        statistics += `**Max members:** ${guild.maximumMembers} \n`;
+        statistics += `**Verified: ${guild.verified ? 'yes' : 'no'} \n`;
```
Comment (JulianVennen) [inline src/commands/utility/serverinfo.js:26]:
```suggestion
        statistics += `**Verified:** ${guild.verified ? 'yes' : 'no'} \n`;
```

File: src/commands/utility/serverinfo.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    static description = 'Show the servers info';
+
+    async execute() {
+        const guild = this.message.guild;
+        
+        let generic = '';
+        generic += `**Owner:** ${guild.owner.user.username} \n`;
+        generic += `**Owner ID:** ${guild.ownerID} \n`;
+        generic += `**Created:** ${guild.createdAt.toDateString()} \n`;
+        generic += `**Region:** ${guild.region} \n`;
```
Comment (JulianVennen) [inline src/commands/utility/serverinfo.js:20]: The region is fully lowercase which looks a bit odd.
Maybe convert it to upper case?

File: src/commands/utility/serverinfo.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        generic += `**Guild ID:** ${guild.id} \n`;
+        
+        let statistics = '';
+        statistics += `**Members:** ${guild.memberCount} \n`;
+        statistics += `**Max members:** ${guild.maximumMembers} \n`;
+        statistics += `**Verified: ${guild.verified ? 'yes' : 'no'} \n`;
+        statistics += `**Partnered:** ${guild.partnered ? 'yes' : 'no'} \n`;
+        
+        let embed = new Discord.MessageEmbed()
```
Comment (JulianVennen) [inline src/commands/utility/serverinfo.js:29]:
```suggestion
        const embed = new Discord.MessageEmbed()
```

**Review by JulianVennen — CHANGES_REQUESTED** [review]

File: src/commands/utility/serverinfo.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        statistics += `**Members:** ${guild.memberCount} \n`;
+        statistics += `**Max members:** ${guild.maximumMembers} \n`;
+        statistics += `**Verified:** ${guild.verified ? 'yes' : 'no'} \n`;
+        statistics += `**Partnered:** ${guild.partnered ? 'yes' : 'no'} \n`;
+        
+        const embed = new Discord.MessageEmbed()
+            .setAuthor(`Info of ${guild.name}`, guild.iconURL())
+            .setThumbnail(guild.iconURL())
+            .setFooter(`Command executed by ${this.message.author.username}`)
+            .setTimestamp();
```
Comment (JulianVennen) [inline src/commands/utility/serverinfo.js:33]:
```suggestion
            .setTimestamp()
```

**Review by JulianVennen — CHANGES_REQUESTED** [review]

**Review by JulianVennen — APPROVED** [review]

**Review by matthi4s — APPROVED** [review]

**Pull request merged**